### DISCUSSION TITLE
Alist 可以 下载整个目录吗？

### DISCUSSION THREAD

**Discussion by lesca** [AlistGo/alist#3749] [General]
我一般都是通过`aria2`下载的，文件夹选中后发送到 `aria2` 没有产生任何下载任务；文件可以批量选择，发送到 aria2 后能正常下载。

现在的问题是，有的文件夹**嵌套**了很多子文件夹，如果只下载文件就会**失去目录结构**。请问是否有办法下载目录？谢谢！

![image](https://user-images.githubusercontent.com/5976219/223294962-672f54a2-2d9a-4f39-a1af-a50dd88e5b3d.png)

**Comment by peeweep** [comment]
+1, 有同样的疑问。

**Comment by pongfcnkl** [comment]
开webdav下载

**Reply by ypq123456789** [reply]
具体怎么操作呢大哥？小白求教

**Reply by pongfcnkl** [reply]
> 具体怎么操作呢大哥？小白求教

看文档

**Comment by anwen-anyi** [comment]
~~暂时支持用Aria2下载文件夹，~~
~~要么用楼上老哥说的用webdav方式拖动出来~~
~~要么自己整理目录~~

-----
已经支持，需要最低 v3.18.0

**Reply by icezxf** [reply]
怎么用aria2下载文件夹啊？是暂时不支持吧？

**Reply by anwen-anyi** [reply]
> 怎么用aria2下载文件夹啊？是暂时不支持吧？

现在支持了 需要最新版本

**Comment by Leexunhuan743** [comment]
把你下载位置也挂载在alist上，然后用复制即可

**Reply by lvls921** [reply]
niubi

**Reply by seepine** [reply]
再改配置 `copy.workers`，起飞

**Reply by llg38456** [reply]
请问 copy.workers 怎么配置？

**Reply by seepine** [reply]
config.json 配置文件，tasks.copy.workers

**Comment by mapleshadow** [comment]
看官方最新文档，有专门一篇章教你怎么用webdav

**Comment by MichaelLiuCY** [comment]
aria2 下载整个文件夹，如果里面文件很多，最终下载后，有些文件会丢失，只有几B大小
这个是什么问题，或者有什么其他解决方案；测试夸克和115都会
